src/routes/dashboard/fragment-wizard/create-fragment-modal.tsx

```diff
@@ -68,6 +83,10 @@ export const CreateFragmentModal = (props: CreateFragmentModalProps) => {
 		navigate(`/edit/${generatedFragment.id}`);
 	};
 
+	const handleChange = (x: CreateOptions, radix: any) => {
+		setSelectedCreateOption(parseInt(CreateOptions[x], radix));
```

Comment:
HandleChange is getting the value so why not just pass the value (x) in this case? 🤔 

Why is the second parameter called `radix`? Doesn't it return the `name` that we passed in to the TileGroup? Then it is being passed in the parseInt function as a base? I don't think we need a `radix` parameter since we only want the value here. 🤔